modules/rubiconAnalyticsAdapter.js

```diff
@@ -224,6 +225,8 @@ function sendMessage(auctionId, bidWonId) {
           'dealsEnforced', () => utils.deepAccess(auctionCache.floorData, 'enforcements.floorDeals'),
           'skipRate',
           'fetchStatus',
+          'floorMin',
+          'floorRuleValue',
```

Comment:
`floorRuleValue` does not make sense to be in the top level `floorData` object of the message. It is a per `bidResponse`

I think this was a typo in the requirements haha.